retroshare-gui/src/gui/MainWindow.cpp

```diff
@@ -677,12 +677,19 @@ void MainWindow::updateTrayCombine()
     updateFriends();
 }
 
+void MainWindow::toggleStatusToolTip(bool toggle){
```

Comment:
toggle parameter is unused, i think this function should only set the tooltip if toggle == true